package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stream",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
```suggestion
  "version": "0.0.2",
```

Reply:
Why 0.0.3? Seems to me 0.0.2 is the latest: https://www.npmjs.com/package/stream?activeTab=versions